sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -96,6 +96,27 @@ case class CatalogStorageFormat(
     }
     map
   }
+
+  def toJsonLinkedHashMap: mutable.LinkedHashMap[String, String] = {
```

Comment:
can we highlight the difference between this and `toLinkedHashMap`? More fields are included?